bl/drawable/src/drawable.cpp

```diff
@@ -1,11 +1,11 @@
 #include "drawable.hpp"
 
-const std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position>
-Drawable::getChildDrawables() const {
+std::unordered_map<std::shared_ptr<Drawable>, Drawable::Position> Drawable::getChildDrawables()
```

Comment:
i'd say it should return const reference to unordeder_map - this way it will be faster and more memory efficient

Reply:
fixed